### DISCUSSION TITLE
GraphQl API for NPM Registry

### DISCUSSION THREAD

**Discussion by Ph0tonic** [npm/feedback#649] [Registry] [answered]
Hello,

I was wondering whether a GraphQL API was planned as it might help fetch the exact wanted data instead of having to retrieve the full package information. Furthermore Github already propose such an API and I think that it could greatly improve the retrieval of specific data.

Thanks

**Comment by MylesBorins** [comment, marked as answer]
Hey @Ph0tonic . At this time we are not exploring a GraphQL api for the npm registry. Thanks for the suggestion 👍🏼
